packages/office-ui-fabric-react/src/components/Sticky/Sticky.tsx

```diff
@@ -115,6 +115,13 @@ export class Sticky extends BaseComponent<IStickyProps, IStickyState> {
   }
 
   public shouldComponentUpdate(nextProps: IStickyProps, nextState: IStickyState): boolean {
+    function isHeightOrWidthDifferent(a: RefObject<HTMLElement>, b: RefObject<HTMLDivElement>): boolean {
```

Comment:
Oops, I merged too soon. This should be defined not in the function, but rather in the file scope and underscored. I believe this function may get recreated on each call.